Sources/FuzzilliCli/main.swift

```diff
@@ -368,6 +370,35 @@ func makeFuzzer(with configuration: Configuration) -> Fuzzer {
     // A script runner to execute JavaScript code in an instrumented JS engine.
     let runner = REPRL(executable: jsShellPath, processArguments: jsShellArguments, processEnvironment: profile.processEnv, maxExecsBeforeRespawn: profile.maxExecsBeforeRespawn)
 
+    /// The mutation fuzzer responsible for mutating programs from the corpus and evaluating the outcome.
+    let disabledMutators = Set(profile.disabledMutators)
+    var mutators = WeightedList([
+        (ExplorationMutator(),              3),
+        (CodeGenMutator(),                  2),
+        (SpliceMutator(),                   2),
+        (ProbingMutator(),                  2),
+        (InputMutator(isTypeAware: false),  2),
+        (InputMutator(isTypeAware: true),   1),
+        // Can be enabled for experimental use, ConcatMutator is a limited version of CombineMutator
+        // (ConcatMutator(),                1),
+        (OperationMutator(),                1),
+        (CombineMutator(),                  1),
+        // Include this once it does more than just remove unneeded try-catch
+        // (FixupMutator()),                1),
+    ])
+    let mutatorsSet = Set(mutators.map {String(describing: type(of: $0))})
+    if !disabledMutators.isSubset(of: mutatorsSet) {
+        configError("The following mutators in " + profileName + " profile's disabledMutators do not exist: " + disabledMutators.subtracting(_: mutatorsSet).description + ". Please check and remove them from your profile configuration.")
+    }
```

Comment:
Currently, if there's any mutators in the specified profile's `disabledMutators` that aren't in `mutators`, then we print this message and exit.

Do you think we should exit after informing the user or let Fuzzilli keep executing (i.e. switching from using `configError` to `logger.info`)?

Reply:
I think this should be a `configError`